The number of milvus operation failure is always zero in chaos test

<!-- Please state your issue using the following template and, most importantly, in English. -->

#### Steps/Code to reproduce:
1. When counting the `succ_rate` in chaos test, it is found that the `fail` num is always 0.
```
    def keep_running(self):
        while self._running is True:
            search_vec = cf.gen_vectors(5, ct.default_dim)
            _, result = self.c_wrap.search(
                                data=search_vec,
                                anns_field=ct.default_float_vec_field_name,
                                param={"nprobe": 32},
                                limit=1, timeout=timeout
                            )
            if result:
                self._succ += 1
            else:
                self._fail += 1
            sleep(constants.WAIT_PER_OP / 10)
```
2. Because we count the `fail` according to the `result` actually returned by `ResponseChecker`. However, when we expect that the successful request fails, it returns an exception before calling `ResponseChecker` and getting the `result`. You can simply understand it based on the following example. In a sense, the `check_result` returned by `ResponseChecker` has no meaning.
```
    def test_collection(self):
        self._connect()
        c_name = cf.gen_unique_str(prefix)
        res, check = self.collection_wrap.init_collection(c_name)
        log.debug(f'res: {res}')
        log.debug(f'check: {check}')
```
```
[2021-08-17 16:44:41 - DEBUG - ci_test]: (api_request)  : [Collection] args: ['collection_y5KefpWL', None], kwargs: {} (api_request.py:44)
[2021-08-17 16:44:41 - ERROR - ci_test]: Traceback (most recent call last):
  File "/home/zong/zong/projects/milvus/tests20/python_client/utils/api_request.py", line 18, in inner_wrapper
    res = func(*args, **kwargs)
  File "/home/zong/zong/projects/milvus/tests20/python_client/utils/api_request.py", line 45, in api_request
    return func(*arg, **kwargs)
  File "/home/zong/zong/.virtualenvs/milvus/lib/python3.8/site-packages/pymilvus_orm/collection.py", line 131, in __init__
    raise SchemaNotReadyException(0, ExceptionsMessage.NoSchema)
pymilvus_orm.exceptions.SchemaNotReadyException: <SchemaNotReadyException: (code=0, message=Should be passed into the schema.)>
 (api_request.py:26)
[2021-08-17 16:44:41 - ERROR - ci_test]: (api_response) : <SchemaNotReadyException: (code=0, message=Should be passed into the schema.)> (api_request.py:27)

        self._connect()
        c_name = cf.gen_unique_str(prefix)
>       res, check = self.collection_wrap.init_collection(c_name)

testcases/test_collection.py:57: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
base/collection_wrapper.py:23: in init_collection
    check_result = ResponseChecker(res, func_name, check_task, check_items, is_succ,
check/func_check.py:31: in run
    result = self.assert_succ(self.succ, True)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

actual = False, expect = True

    @staticmethod
    def assert_succ(actual, expect):
>       assert actual is expect
E       AssertionError

check/func_check.py:67: AssertionError
```

3. How to fix: The statistical condition for `succ` and `fail` is whether to return an exception rather than `result`

#### Expected result:


#### Actual results:



#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others: 

#### Configuration file:


#### Additional context:

--- a/tests20/python_client/chaos/checker.py
+++ b/tests20/python_client/chaos/checker.py
@@ -7,6 +7,8 @@ from base.collection_wrapper import ApiCollectionWrapper
 from common import common_func as cf
 from common import common_type as ct
 import constants
+
+from common.common_type import CheckTasks
 from utils.util_log import test_log as log
 
 
@@ -63,7 +65,7 @@ class SearchChecker(Checker):
                                 data=search_vec,
                                 anns_field=ct.default_float_vec_field_name,
                                 param={"nprobe": 32},
-                                limit=1, timeout=timeout
+                                limit=1, timeout=timeout, check_task=CheckTasks.check_nothing
                             )
             if result:
                 self._succ += 1
@@ -82,7 +84,7 @@ class InsertFlushChecker(Checker):
         while self._running:
             _, insert_result = \
                 self.c_wrap.insert(data=cf.gen_default_list_data(nb=constants.DELTA_PER_INS),
-                                   timeout=timeout)
+                                   timeout=timeout, check_task=CheckTasks.check_nothing)
             if not self._flush:
                 if insert_result:
                     self._succ += 1
@@ -106,7 +108,7 @@ class CreateChecker(Checker):
             _, result = self.c_wrap.init_collection(
                                     name=cf.gen_unique_str("CreateChecker_"),
                                     schema=cf.gen_default_collection_schema(),
-                                    timeout=timeout
+                                    timeout=timeout, check_task=CheckTasks.check_nothing
                                 )
             if result:
                 self._succ += 1
@@ -128,7 +130,7 @@ class IndexChecker(Checker):
             _, result = self.c_wrap.create_index(ct.default_float_vec_field_name,
                                                  constants.DEFAULT_INDEX_PARAM,
                                                  name=cf.gen_unique_str('index_'),
-                                                 timeout=timeout)
+                                                 timeout=timeout, check_task=CheckTasks.check_nothing)
             if result:
                 self._succ += 1
                 self.c_wrap.drop_index(timeout=timeout)
@@ -147,7 +149,7 @@ class QueryChecker(Checker):
             for _ in range(5):
                 int_values.append(randint(0, constants.ENTITIES_FOR_SEARCH))
             term_expr = f'{ct.default_int64_field_name} in {int_values}'
-            _, result = self.c_wrap.query(term_expr, timeout=timeout)
+            _, result = self.c_wrap.query(term_expr, timeout=timeout, check_task=CheckTasks.check_nothing)
             if result:
                 self._succ += 1
             else:

--- a/tests20/python_client/check/func_check.py
+++ b/tests20/python_client/check/func_check.py
@@ -34,6 +34,9 @@ class ResponseChecker:
             # Interface return error code and error message check
             result = self.assert_exception(self.response, self.succ, self.check_items)
 
+        elif self.check_task == CheckTasks.check_nothing:
+            return self.succ
+
         elif self.check_task == CheckTasks.ccr:
             # Connection interface response check
             result = self.check_value_equal(self.response, self.func_name, self.check_items)

